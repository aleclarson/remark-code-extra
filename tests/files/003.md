```not-skipped
simple code block
```

```skipped
language code block
```