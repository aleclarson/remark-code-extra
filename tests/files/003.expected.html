simple code block
language code block
